docs/run/quantum-serverless.mdx

```diff
@@ -57,11 +57,19 @@ pm = generate_preset_pass_manager(optimization_level=3, backend=backend)
 target_circuit = pm.run(abstract_circuit)
 target_observable = observable.apply_layout(target_circuit.layout)
 
-# Step 3: Execute the target circuit
+# Step 3: Execute the target circuit on Estimator V1
+# from qiskit_ibm_runtime import Estimator
+# estimator = Estimator(backend)
+# job = estimator.run(target_circuit, target_observable)
+# result = job.result()
+
+# Step 3: Execute the target circuit on Estimator V2
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
 estimator = Estimator(backend)
-job = estimator.run(target_circuit, target_observable)
+job = estimator.run([(target_circuit, target_observable)])
 result = job.result()
 
+
 # Step 4: Postprocess the results
 print(result)
 
```

Comment:
I can't leave a comment on line 78, but this step should be changed to something like

```
# Step 4: Postprocess the Estimator V1 results
# print(result)
# save results of program execution
# note: saved items must be serializable
# save_result(result.values)

# Step 4: Postprocess the Estimator V2 results
print(result)
# save results of program execution
# note: saved items must be serializable
save_result(result[0].data.evs)
```